e2e-tests/production-runtime/cypress/plugins/index.js

```diff
@@ -5,18 +5,20 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 
-  if (process.env.CYPRESS_CONNECTION_TYPE) {
-    on(`before:browser:launch`, (browser = {}, args) => {
-      if (
-        browser.name === `chrome` &&
-        process.env.CYPRESS_CONNECTION_TYPE === `slow`
-      ) {
+  on(`before:browser:launch`, (browser = {}, args) => {
+    if (browser.name === `chrome`) {
+      if (process.env.CYPRESS_CONNECTION_TYPE === `slow`) {
         args.push(`--force-effective-connection-type=2G`)
       }
 
-      return args
-    })
-  }
+      const index = args.indexOf(`--disable-gpu`)
```

Comment:
Why is this necessary?

Reply:
This isn't related to tests failing but makes it easier to run on Linux - otherwise Cypress uses software rendering by default so the UI is really slow

Reply:
so it takes me twice as long to look through the console etc